app/models/notification.rb

```diff
@@ -75,7 +75,7 @@ def self.ensure_consistency!
     DB.exec(<<~SQL)
       DELETE
         FROM notifications n
-       WHERE high_priority
+       WHERE high_priority AND notification_type <> #{types[:chat_mention]}
```

Comment:
When interpolating strings, I always prefer to add a `to_i` as a defensive way to avoid injection.